examples/blog/api.ttl

```diff
@@ -10,7 +10,6 @@
 @prefix schema: <http://schema.org/> .
 
 <Blog> a hydra:Class;
-  hydra:search <blog#search>;
```

Comment:
As we discussed `hydra:search` is a link property and belongs to the representations